docs/ops/opset9.md

```diff
@@ -105,7 +105,7 @@ declared in `namespace opset9`.
 * [Mish](activation/Mish_4.md)
 * [Mod](arithmetic/Mod_1.md)
 * [MVN](normalization/MVN_6.md)
-* [MulticlassNMS](sort/MulticlassNMS_8.md)
+* [MulticlassNMS](sort/MulticlassNMS_9.md)
```

Comment:
Can we please rename this document to `MulticlassNonMaxSuppression_9.md` because usually we call the document and the operation similarly?

Reply:
done